docs/rules/max-attributes-per-line.md

```diff
@@ -0,0 +1,88 @@
+# Define the number of attributes allows per line (max-attributes-per-line)
+
+Limits the maximum number of props per line to improve readability
+
+
+## Rule Details
+
+This rule aims to enforce a number of props per line in templates. It checks all the elements in a template and verifies that the number of props per line does not exceed the defined maximum.
+A prop is considered to be in a new line when there is a line break between two props.
+
+The default is one prop per line
+
+Examples of **incorrect** code for this rule:
+
+```js
+
+<component foo="1" bar="3"></component>
```

Comment:
I think default should be:
- singleline: 3
- multiline: 1

Then examples could look like this:
*Incorrect*:
```
<component lorem="1" ipsum="2" dolor="3" sit="4">
</component>

<component
  lorem="1" ipsum="2"
  dolor="3"
  sit="4"
>
</component>
```

*Correct*:
```
<component lorem="1" ipsum="2" dolor="3">
</component>

<component
  lorem="1"
  ipsum="2"
  dolor="3"
  sit="4"
>
</component>
```

We should probably add additional option for specifying if we accept attribute in first line or not too, maybe something like `firstline: 1`, that would make it possible to have one property in the first line, or more if specified. WDYT?